contracts/libs/RealMath.sol

```diff
@@ -13,17 +14,17 @@ pragma solidity ^0.5.17;
 
 library RealMath {
 
-    /**
+    /*
```

Comment:
I think it should be` //` and not `/*`
https://solidity.readthedocs.io/en/v0.6.10/layout-of-source-files.html#comments

Reply:
But the doc you send said multi-line comments should be done with `/**/`

Reply:
though this is not multi comment ..

Reply:
ok, I don't think this really matters but anyway I changed that